subsys/bootloader/c_runtime_setup/zephyr_dummy_interrupt_handlers.c

```diff
@@ -26,8 +26,14 @@ void __secure_fault(void) ALIAS("dummy_handler");
 void __svc(void) ALIAS("dummy_handler");
 void __pendsv(void) ALIAS("dummy_handler");
 void __sys_tick(void) ALIAS("dummy_handler");
+void z_clock_isr(void) ALIAS("dummy_handler");
```

Comment:
Could you put this in a separate commit?

Reply:
Sure